src/Symfony/Component/Notifier/Bridge/OvhCloud/OvhCloudTransport.php

```diff
@@ -75,20 +75,22 @@ protected function doSend(MessageInterface $message): void
         $now = time() + $this->calculateTimeDelta();
         $headers['X-Ovh-Application'] = $this->applicationKey;
         $headers['X-Ovh-Timestamp'] = $now;
+        $headers['Content-Type'] = 'application/json';
 
-        $toSign = $this->applicationSecret.'+'.$this->consumerKey.'+POST+'.$endpoint.'+'.json_encode($content, \JSON_UNESCAPED_SLASHES).'+'.$now;
+        $body = json_encode($content, \JSON_UNESCAPED_SLASHES);
+        $toSign = $this->applicationSecret.'+'.$this->consumerKey.'+POST+'.$endpoint.'+'.$body.'+'.$now;
         $headers['X-Ovh-Consumer'] = $this->consumerKey;
         $headers['X-Ovh-Signature'] = '$1$'.sha1($toSign);
 
         $response = $this->client->request('POST', $endpoint, [
             'headers' => $headers,
-            'json' => $content,
+            'body' => $body,
         ]);
 
         if (200 !== $response->getStatusCode()) {
             $error = $response->toArray(false);
 
-            throw new TransportException(sprintf('Unable to send the SMS: %s.', $error['message']), $response);
+            throw new TransportException(sprintf('Unable to send the SMS: "%s".', $error['message']), $response);
```

Comment:
this should be reverted, even if fabbot says it should be updated, that's a false-positive

Reply:
this could be switched to using concatenation rather than `sprintf` as done elsewhere to avoid the false positive.